libs/api-model-bear/api/api-model-bear.api.md

```diff
@@ -1044,6 +1044,10 @@ export namespace GdcExport {
     }
     // (undocumented)
     export interface IExportResponse {
+        // (undocumented)
+        fileName?: string;
+        // (undocumented)
+        objectUrl?: string;
```

Comment:
What is difference between  objectUrl and uri?

Reply:
`uri` points to our API endpoint from which you can fetch the result (again, as it was already fetched once when polling resolved). `objectUrl` points to blob of data attached to the current `window` instance. The blob contains data from the polling result. The Object URL looks like this: `blob:https://localhost:8443/51b3d300-65ba-41e0-9bd2-7a23a75c571b` for example. Also, it is necessary to revoke it when you do not needed to release the blob data from memory and `window` instance. 

This could be an issue of this solution as it will be created every time you create export and you must not forget to revoke it, otherwise you will gather memory. Not sure if I should not do another method for exporting with this Object URL? Possibly put somewhere some `try finaly` block to catch possible errors and revoke the URL if it would not come to the actual download? I'm not sure that could happen however. Once you parse response and create blob, there are not many places when the logic could fail IMHO.